rtgui/previewwindow.cc

```diff
@@ -199,6 +199,8 @@ bool PreviewWindow::on_motion_notify_event (GdkEventMotion* event)
 
         if (isMoving) {
             mainCropWin->remoteMove ((event->x - press_x) / zoom, (event->y - press_y) / zoom);
```

Comment:
Just out of curiosity: Was this a bug? Could I notice that this was missing in the current `master` branch?